packages/contracts/contracts/core/DAO.sol

```diff
@@ -233,4 +237,13 @@ contract DAO is IDAO, Initializable, UUPSUpgradeable, PermissionManager, ERC1271
 
         emit TrustedForwarderSet(_trustedForwarder);
     }
+
+    /// @inheritdoc IDAO
+    function registerStandardAndCallback(
+        bytes4 _interfaceId,
+        bytes4 _callbackSig,
+        bytes4 _magicNumber
+    ) external override auth(address(this), REGISTER_STANDARD_PERMISSION_ID) {
+        _registerStandardAndCallback(_interfaceId, _callbackSig, _magicNumber);
+    }
```

Comment:
I still don't understand in which situation we need AdaptiveERC165 and its callback functionality.

Why does our DAO want to add more interfaces? 

If there is a reason, I would just use OZs `ERC165Storage` (see https://docs.openzeppelin.com/contracts/4.x/api/utils#ERC165Storage and https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/introspection/ERC165Storage.sol) that can register multiple interfaces and remove `AdaptiveERC165`.
We would need to add a `register` function here in this case.

If the DAO doesn't need this, `DAO` should just inherit from OZs `ERC165` and have only one interface.

Reply:
Resolved, by discussions with @novaknole and merge of https://github.com/aragon/core/pull/112